apps/studio/prisma/schema.prisma

```diff
@@ -66,13 +68,20 @@ model Resource {
 
   state ResourceState? @default(Draft)
   type  ResourceType
+
+  createdAt DateTime @default(now())
+  updatedAt DateTime @default(now()) @updatedAt
```

Comment:
the @updatedAt tag is to auto inject this on Prisma ORM level right? but since we are using kysely how does this help?

Reply:
Yeah there is no functional difference here since we are not using the prisma client and we are only using prisma for the modelling, hence the need to introduce the triggers on the DB level unfortunately.